src/test/resources/com/arpnetworking/logback/StenoEncoderTest.testEncodeArrayWithException.json

```diff
@@ -1 +1 @@
-{"time":"1970-01-01T00:00:00.000Z","name":"logEvent","level":"info","data":{},"exception":{"type":"java.lang.NullPointerException","message":"npe!","backtrace":[],"data":{}},"context":{"host":"<HOST>","processId":"<PROCESS_ID>","threadId":"<THREAD_ID>"},"id":"<ID>","version":"0"}
+{"time":"1970-01-01T00:00:00.000Z","name":"logEvent","level":"info","data":{},"exception":{"type":"java.lang.NullPointerException","message":"npe!","backtrace":[],"data":{"_id":"<ID>","_class":"java.lang.NullPointerException"}},"context":{"host":"<HOST>","processId":"<PROCESS_ID>","threadId":"<THREAD_ID>"},"id":"<ID>","version":"0"}
```

Comment:
In that case, u would almost argue that the whole exception block should be the exception, then.  I understand why you did things the way you did and agree.  It seems like special casing the exception part was the right way to go, but that you didn't go quite far enough because of how the data is duplicated.  I almost want you to provide a default logging format for Throwable that removes the class, message, and stack trace. Can we even do that without breaking everything?